pages/blog/blog-post.js

```diff
@@ -314,31 +314,29 @@ const BlogPostNav = ({
   nextPostRoute,
   nextPostText,
 }) => (
-  <PageSection>
-    <div className="row">
-      <div className="offset-lg-2 col-lg-8">
-        <HorizontalRule />
-
-        <NavContainer>
-          <Link route={previousPostRoute} passHref>
-            <NavButton type="secondary">
-              <span className="longNavText">{previousPostText}</span>
-              <span className="shortNavText">{'<'}</span>
-            </NavButton>
-          </Link>
-          <Link route={routes.Blog} passHref>
-            <NavButton type="secondary">{blogHomeText}</NavButton>
-          </Link>
-          <Link route={nextPostRoute} passHref>
-            <NavButton type="secondary">
-              <span className="longNavText">{nextPostText}</span>
-              <span className="shortNavText">{'>'}</span>
-            </NavButton>
-          </Link>
-        </NavContainer>
-      </div>
+  <div className="row">
```

Comment:
You are having issues with container since you removed the `PageSection`. If you still want container without paddings you can just use `<div className="container">your content</div>`